USA/FL/Bay/Court/scraper/tests/test_ScraperUtils.py

```diff
@@ -57,12 +59,22 @@ def test_parse_plea_type_not_guilty(self):
         assert ScraperUtils.parse_plea_type(plea2) == 'Not Guilty'
         assert ScraperUtils.parse_plea_type(plea3) == 'Not Guilty'
 
+    @flagsaver.flagsaver(collect_pii=True)
     def test_parse_name(self):
         name1 = "DOE, JANE EMILY"
         name2 = "DOE, JOHN"
         assert ScraperUtils.parse_name(name1) == ('JANE', 'EMILY', 'DOE')
         assert ScraperUtils.parse_name(name2) == ('JOHN', None, 'DOE')
 
+    @flagsaver.flagsaver(collect_pii=False)
+    def test_parse_name_redacted(self):
+        name1 = "DOE, JANE EMILY"
+        name2 = "DOE, JOHN"
+        assert ScraperUtils.parse_name(name1) == (
+                '[redacted]', '[redacted]', '[redacted]')
+        assert ScraperUtils.parse_name(name2) == (
+                '[redacted]', '[redacted]', '[redacted]')
```

Comment:
I like this. I agree that it's more transparent to write '[redacted]' than to leave the fields blank.